shell/platform/darwin/graphics/FlutterDarwinExternalTextureMetal.mm

```diff
@@ -220,10 +255,27 @@ - (void)onTextureUnregistered {
   id<MTLTexture> rgbaTex = CVMetalTextureGetTexture(metalTexture);
   CVBufferRelease(metalTexture);
 
-  return [FlutterDarwinExternalTextureSkImageWrapper wrapRGBATexture:rgbaTex
-                                                           grContext:grContext
-                                                               width:textureSize.width()
-                                                              height:textureSize.height()];
+#if IMPELLER_SUPPORTS_RENDERING
```

Comment:
Is the ifdef check necessary at all on Darwin TUs? AFAIK, only Fuchsia has this off.

Reply:
Done.